test/functional/config/utils.go

```diff
@@ -29,15 +39,39 @@ type TestUtils struct {
 	PollingPeriod time.Duration
 	Namespace     string
 	Verbose       bool
+	Client        client.Client
 }
 
-func CreateDefaultTestUtils() *TestUtils {
+func init() {
+	utils.Must(resources.Register(corev1.SchemeBuilder))
+	utils.Must(resources.Register(apiextensionsv1.SchemeBuilder))
+	utils.Must(resources.Register(v1alpha1.SchemeBuilder))
+	utils.Must(resources.Register(extensionsv1alpha.SchemeBuilder))
+	utils.Must(resources.Register(dnsapi.SchemeBuilder))
+}
+
+func CreateDefaultTestUtils() (*TestUtils, error) {
+	kubeconfig := os.Getenv("KUBECONFIG")
+	if kubeconfig == "" {
+		return nil, fmt.Errorf("KUBECONFIG not set")
+	}
+	restConfig, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	if err != nil {
+		return nil, fmt.Errorf("Error loading kubeconfig from %s: %v\n", kubeconfig, err)
```

Comment:
Minor detail: Can the `\n` at the end of the error message be removed?
```suggestion
		return nil, fmt.Errorf("Error loading kubeconfig from %s: %v", kubeconfig, err)
```